src/server/services/move_servo.py

```diff
@@ -1,5 +1,33 @@
 import logging
+import multiprocessing
```

Comment:
please use threads, not processes
Or maybe even `concurrent.futures` module https://docs.python.org/3/library/concurrent.futures.html